README.md

```diff
@@ -1,22 +1,19 @@
 # Xùn
 
 ```
-████████████████████████████████
-████████████████████████████████
+█████████████
 
+█████████████
 
-████████████████████████████████
-████████████████████████████████
-
-
-██████████████    ██████████████
-██████████████    ██████████████
+█████   █████
 
 https://en.wikipedia.org/wiki/Bagua
 ```
 
 [![equinor](https://circleci.com/gh/equinor/xun.svg?style=shield)](https://app.circleci.com/pipelines/github/equinor/xun)
 
+Distributed functional python framework
```

Comment:
```suggestion
Xun is a distributed and functional Python framework for cluster compute. Rather than focusing on batching jobs, xun is about defining values declaratively.
```